hooks/useLessonUnlocked.ts

```diff
@@ -0,0 +1,50 @@
+'use client'
+
+import { useUser, usePathData } from 'hooks'
+import { chapters } from 'content'
+
+function removeAfterHyphen(str: string): string {
+  return str.split('-')[0]
+}
+
+function getIndex(arr: string[], searchStr: string): number {
```

Comment:
I dont think we need a new function if all its doing is `arr.indexOf(searchStr)`, just use that instead of `getIndex` on line 37